lib/commands/sdk.js

```diff
@@ -1444,18 +1444,21 @@ exports.getReleases = async function getReleases(config, os) {
 		// Paginate through the releases on the repo, and determine which of the assets
 		// is correct for the platform we're running on
 		const releaseData = await octokit.paginate(paginateData);
-		const osRegex = /mobilesdk-\d+\.\d+\.\d+\.GA-(\w+)\.zip/;
+		const osRegex = /mobilesdk-\d+\.\d+\.\d+\.([A-Z)]{2})-(\w+)\.zip/;
 		for (let { assets, name } of releaseData) {
 			const release = assets.find(asset => {
-				const [ , assetOS ] = osRegex.exec(asset.name);
+				const [ , versionType, assetOS ] = osRegex.exec(asset.name);
```

Comment:
```suggestion
				const [ , , assetOS ] = osRegex.exec(asset.name);
```
versionType is not used.